src/GetOrCreateImage/GetOrCreateImage.js

```diff
@@ -26,6 +26,7 @@ const GetOrCreateImage = async event => {
   } = event.Records[0]
 
   if (!['403', '404'].includes(status)) return response
+  if (!querystring) return response
```

Comment:
Technically, we should be returning an HTTP 400 error (bad request).

Reply:
Actually, this should never evaluate to true according to this line https://github.com/HoraceShmorace/Image-Flex/blob/b02b7415767dc78e0f508aee5f8f45a2f936d006/src/UriToS3Key/UriToS3Key.js#L30